apps/storybook-e2e/src/e2e/sections/header/vl-header.stories.cy.ts

```diff
@@ -18,4 +18,34 @@ describe('story vl-header', () => {
         cy.createStubForEvent('vl-header', 'ready');
         cy.get('@ready').should('have.been.calledOnce');
     });
+
+    describe('vl-header - container <#header__container>', () => {
```

Comment:
mss is `vl-header - container` voldoende

Reply:
opgelost